resources/views/dashboard/controllers/profile.blade.php

```diff
@@ -129,27 +129,31 @@
                 <table class="table table-bordered text-center">
                     <thead>
                         <tr>
-                            <th scope="col">Date</th>
-                            <th scope="col">Start Time</th>
+                            <th scope="col">Date/time</th>
                             <th scope="col">Lesson Type</th>
                             <th scope="col">Instructor/Mentor</th>
                             <th scope="col">View/Modify</th>
                         </tr>
                     </thead>
                     <tbody>
-                        @if(count($ea_appointments) > 0)
-                            @foreach($ea_appointments as $ea_appointment)
+                        @if($appointments_successful)
+                            @if(count($appointments) > 0)
+                                @foreach($appointments as $appointment)
+                                    <tr>
+                                        <td scope="col">{{ \Carbon\Carbon::parse($appointment->session->start)->setTimezone(Auth::User()->timezone) }} {{ Auth::User()->timezone_abbr }}</td>
```

Comment:
@c0repwn3r one comment... can you format this date/time a little cleaner? It is currently displaying as 
2024-11-13 19:00:00 EST - it would be nicer if the date format matched all of the other dates on the profile page in a m/d/Y format, and then just hh:mm without the seconds.